packages/react-core/src/hooks/useDropZone.ts

```diff
@@ -86,17 +87,35 @@ export default function useDropZone({
     event.stopPropagation();
     setDragState('inactive');
     const files = Array.from(event.dataTransfer.files);
-    const { acceptedFiles, rejectedFiles } = filterAllowedFiles<File>(
-      files,
-      acceptedFileTypes
-    );
 
     if (isFunction(_onDrop)) {
       _onDrop(event);
     }
-    if (isFunction(onDropComplete)) {
-      onDropComplete({ acceptedFiles, rejectedFiles });
+
+    // If no DataTransfer items interface, handle as simple files
+    if (!event.dataTransfer.items) {
+      const { acceptedFiles, rejectedFiles } = filterAllowedFiles<File>(
+        files,
+        acceptedFileTypes
+      );
+      if (isFunction(onDropComplete)) {
+        onDropComplete({ acceptedFiles, rejectedFiles });
+      }
+      return;
     }
+
+    // Process items using util
+    processDroppedEntries(Array.from(event.dataTransfer.items)).then(
```

Comment:
Nit: `processDroppedItems` to be more specific that we're dealing with `items`

Reply:
Updated